workflow-report/README.md

```diff
@@ -0,0 +1,45 @@
+# GitHub Action: report failed job to VK Team chat
+This action composes a message about the failed job and sends it to VK Teams chat.
+The message contains the following information:
+* __Job__: name of the failed job with the link to this job
+* __Commit__: hash of the commit started the job with the link to this commit
+* __Branch__: name of the branch with the link to this branch
+* __History__: the link to commit history
+* __Triggered on__: name of the event triggered the job 
+* __Committer__: GitHub login of the commit author
+* __Commit message subject__: subject (first string) of the commit message
+* __Failed steps info__: info about the failed steps (outputs, outcome, conclusion)
+
+# Prepare your workflow
+The action can tell you about the exact steps that failed only if your steps have IDs, example:
```

Comment:
I would replace it to `The action can tell you about the failed steps only if they have IDs`.

Reply:
done